decidim-core/app/helpers/decidim/card_helper.rb

```diff
@@ -10,6 +10,8 @@ module CardHelper
     #
     # Returns an HTML.
     def card_for(model, options = {})
```

Comment:
This method seems to be used for both users (authors) and proposals, but I don't see any logic redirecting the cells to the correct one, what am I missing? 😕

Reply:
as you can see two lines bellow:
https://github.com/decidim/decidim/blob/b9dd085aa188f161c70c0800cf86fc96326b4b23/decidim-core/app/helpers/decidim/card_helper.rb#L15

The redirection logic goes to the `CardCell` https://github.com/decidim/decidim/blob/b9dd085aa188f161c70c0800cf86fc96326b4b23/decidim-core/app/cells/decidim/card_cell.rb#L4

Reply:
Oh, perfect then!